control/sum.go

```diff
@@ -34,25 +34,46 @@ func newSum(config BlockConfig, logger logging.Logger) (Block, error) {
 }
 
 func (b *sum) Next(ctx context.Context, x []*Signal, dt time.Duration) ([]*Signal, bool) {
```

Comment:
The comments are in `controllable.go`, I'd rather not mess too much with the control package until we figure out multi-dimensional signals.

Reply:
I totally agree: I'd rather not mess with this package until it's clear what's supposed to happen. That's why I'm asking for the comment: we're messing with it and I can't figure out what the changes are supposed to do, because the dimensions aren't matching up.